yunohost_project_organization_fr.md

```diff
@@ -189,14 +204,13 @@ Dans ce cas, l'auteur annonce sur le fil de discussion de la PR qu'il souhaite e
 ## Composition des groupes
 
 - Conseil : Bram, ljf, Maniack, Moul, Aleks, frju365, Josue, JimboJoe
-- Core Dev : Aleks, Bram, JimboJoe, Ju, ljf, Moul, opi
-- Apps : Bram, cyp, frju365, JimboJoe, Josue-T, Ju, ljf, Maniack C, Maxime, Moul, Scith, Tostaki
-- Infra : Bram, Ju, Maniack C, Moul, opi
-- Communication
-  - Com : Bram, Moul, korbak, ljf, opi, frju365
-  - Doc : Moul, Theodore
-  - Trad : Jean-Baptiste
-- Distribution : Heyyounow
+- Core : Aleks, Bram, ljf, JimboJoe, Josue, (...?)
+- Apps : Maniack, frju365, JimboJoe, Josue, ljf, (remove : cyp? Maxime? Scith? Tostaki?) (add : Jibec? tituspijean? nicofrand? amnol? Gofannon? ...?)
```

Comment:
I am ready to take this responsibility. I have to start contributing to helpers. This role will fill the gap between using helpers and leaning to code helpers.

Reply:
@anmol26s, for me it's ok, still I would really like to have the point of views of the others. But, I think they're going to be ok too.

By the way, because I see that I've forgotten to mention Jibec, let's the facts talk for me...
https://github.com/YunoHost-Apps/agendav_ynh/issues/28
https://github.com/YunoHost-Apps/agendav_ynh/pull/29
I spend a lot of time fixing his work, that's not really what we expect of a member of this group...

Reply:
I agree with Maniack. For @anmol26s and @tituspijean, I'm ok. they did good work on Flarum and grav, and because this was difficult, I'm totally ok to invite them in Apps Group. 

For @Jibec , one more time, he wanted to leave the project, so in the respect of this, I'm against integrating him in Apps Group (because in fact, we need to have person who works for a long time... and it will not be the case as I understood from @Jibec's post on the forum).

For @nicofrand... I don't know.

For @Gofannon it's ok even if for me it seems to be a bit soon even if I agree totally that he did very good thing with his apps.

Removing @magikcypress and @polytan02 is ok as far as they seems to be not anymore in the project. For others... it's ok too as far as they agree.

Reply:
euh no... he is the owner of Flarum, Owntracks package. He did a good job in flarum package, and even create a Flarum extension to integrate SSO authentification in flarum https://github.com/tituspijean/flarum-ext-auth-ssowat... he is TitusPiJean in the forum ;)...

Reply:
Ok, maybe it's because I'm not using Flarum or Owntracks, but I've never heard of him before.
And that make me think that he should probably be more involved in the daily activity of the apps group before considering including him in it.

Reply:
mh... ok. He already communicate with us on the chat...  I think he is really involved in Apps packaging, perhaps not in Apps Group directly, but it's sure for packaging.

Reply:
Personally I prefer, if you agree with, to stay in the Apps group as I have the borg and borg_server apps, and odoo_ynh, coin_ynh too. Note I have also maintain strut recently, i just wait for validation.

Reply:
I'm not willing to kick you out, it's just that this group, or the current list, seem really meaningless to me.
To be honest, this group is made of 12 members (!!!), for only 4 really active on the real purpose of this group... Maybe we should think about the purpose of this group or its utility, or just forget about it...
Don't know...

Reply:
in fact, if ppl who want to join or stay in the Apps group, don't want to work on helpers, CI or the Official Apps hardly, they can leave. There is no point in being in a group if the work is not done. Moreover ppl can work on apps, even if from outside of the Apps Group, it's not a problem.

I think Apps Group like other group make supervision on work and for that, members of the group must know all the recent work who has been done on Official Apps, CI or helpers. Moreover, the member of the Apps Group is a sort of Example for other packagers too.

By this comment, I want just to say that ppl who want to join or keep being in the Apps Group have to be active from now and be present on quite every official Apps & helpers (at least approving PR).

It's my point of view.

Reply:
To elaborate a bit on why I proposed the people I listed : I naively think about the app group as "people who participate in app packaging in general, e.g. have packaged several apps recently - and therefore might have a relevant opinion about app packaging in general".

I don't know how much and which responsabilities come with it - though my understanding is that it is of course encouraged to help with apps reviews / maintenance (especially official ones) and helpers. (Being in this group can also help discovering these aspects and feeling legitimate to participate)

But agreed that app packaging is such a broad topic that there are indeed at least two groups (or sub groups) which one could call "core" app maintenance & tools (?) and the wider "app packaging community". 

I dunno what do with all this :P

Reply:
I was about to say that I was totally fine about being removed from the apps group because, well, I don't package applications at all.

Then I reread the responsibilities linked to the app group and I realized that I'm working on quite some part of the app (and translation) backoffice/structure, like:

- list_builder
- apps translations (including new behavior that is on my todo list)
- yunorunner 
- action and config panel in the core

So idk :man_shrugging: 

I still don't packages application and barely maintain lektor only for my own needs so I don't feel relevant regarding app packaging itself so I don't feel I'm useful in it but I definitively need a way to communicate with the app group (which today is mainly talking to @maniackcrudelis which is not to optimal but works).

I would be good to have again someone in the translation part but, yeah, for now I'm mainly on the CI.

Reply:
Too soon to integrate the group Apps and not sure if I am willing to. For now, I plan to work on some apps and I cannot give more time that what I am already doing. I will have to reduce that amount too soon.

So wait and see on my side.

Reply:
OK, I'm not necessarily comfortable with the idea of speaking about people here in public, especially when it means sometimes expressing direct judgments on others... :thinking: 
Regarding @Jibec , thanks @frju365 for pointing to the forum: he actually made his farewells in August (with which I'm a little bit sad!).
I approve @maniackcrudelis description of what this group is. I just can't agree right now to be counted as 1 in the statistics, as my contribution time unfortunately tends to be homeopathic... :cry: 
I agree with the proposed removal of @magikcypress, @polytan02, @scith and @mbugeia (unless they voice concern here, of course!).
I agree to welcome @anmol26s in the group :heart: 
@Gofannon and @tituspijean contributions to the apps seem significative to me (thanks for that ! :heart:) and I'm totally open to welcome them in a short term.